doc/enterprise/flight_recorder.rst

```diff
@@ -44,175 +48,16 @@ Tarantool continues writing to the existing ``current.ttfr`` file after restart.
-
-.. confval:: flightrec_logs_log_level
-
-    Specifies the level of detail the log has.
-    You can learn more about log levels from the :ref:`log_level <cfg_logging-log_level>`
-    option description.
-    Note that the ``flightrec_logs_log_level`` value might differ from ``log_level``.
-
-    | Type: integer
-    | Default: 6
-    | Environment variable: TT_FLIGHTREC_LOGS_LOG_LEVEL
-
-
-.. _config-metrics:
-
-Metrics
-~~~~~~~
-
-This section describes the flight recorder settings related to collecting
-:ref:`metrics <metrics-reference>`.
-
-* :ref:`flightrec_metrics_period <cfg_flightrec_metrics_period>`
-* :ref:`flightrec_metrics_interval <cfg_flightrec_metrics_interval>`
-
-.. _cfg_flightrec_metrics_period:
-
-.. confval:: flightrec_metrics_period
-
-    Specifies the time period (in seconds) that defines how long metrics are stored from the moment of dump.
-    So, this value defines how much historical metrics data is collected up to the moment of crash.
-    The frequency of metric dumps is defined by :ref:`flightrec_metrics_interval <cfg_flightrec_metrics_interval>`.
-
-    | Type: integer
-    | Default: 180
-    | Environment variable: TT_FLIGHTREC_METRICS_PERIOD
-
-
-.. _cfg_flightrec_metrics_interval:
-
-.. confval:: flightrec_metrics_interval
-
-    Specifies the time interval (in seconds) that defines the frequency of dumping metrics.
-    This value shouldn't exceed :ref:`flightrec_metrics_period <cfg_flightrec_metrics_period>`.
-
-    | Type: number
-    | Default: 1.0
-    | Minimum: 0.001
-    | Environment variable: TT_FLIGHTREC_METRICS_INTERVAL
-
-.. NOTE::
-
-    Given that the average size of a metrics entry is 2 kB,
-    you can estimate the size of the metrics storage as follows:
-
-    .. code-block:: console
-
-        (flightrec_metrics_period / flightrec_metrics_interval) * 2 kB
-
-
-
-.. _config-requests:
-
-Requests
-~~~~~~~~
-
-This section lists the flight recorder settings related to
-storing the :ref:`request and response <internals-requests_responses>` data.
-
-* :ref:`flightrec_requests_size <cfg_flightrec_requests_size>`
-* :ref:`flightrec_requests_max_req_size <cfg_flightrec_requests_max_req_size>`
-* :ref:`flightrec_requests_max_res_size <cfg_flightrec_requests_max_res_size>`
-
-.. _cfg_flightrec_requests_size:
-
-.. confval:: flightrec_requests_size
-
-    Specifies the size (in bytes) of storage for the request and response data.
-    You can set this parameter to ``0`` to disable a storage of requests and responses.
-
-    | Type: integer
-    | Default: 10485760
-    | Environment variable: TT_FLIGHTREC_REQUESTS_SIZE
-
-
-
-.. _cfg_flightrec_requests_max_req_size:
-
-.. confval:: flightrec_requests_max_req_size
-
-    Specifies the maximum size (in bytes) of a request entry.
-    A request entry is truncated if this size is exceeded.
-
-    | Type: integer
-    | Default: 16384
-    | Environment variable: TT_FLIGHTREC_REQUESTS_MAX_REQ_SIZE
-
+Reading flight recordings
```

Comment:
Let's drop this section and document this API separately. The current Tarantool help already contains a hot mix of how-tos, what's new, and other types of info in specific pages.